keyboards/handwired/dactyl_manuform/4x5/keymaps/via/keymap.c

```diff
@@ -0,0 +1,72 @@
+    _BASE,
+    _RAISE,
+    _LOWER,
+};
+
+#define SFT_ESC  SFT_T(KC_ESC)
+#define CTL_BSPC CTL_T(KC_BSPC)
+#define ALT_SPC  ALT_T(KC_SPC)
+#define SFT_ENT  SFT_T(KC_ENT)
+
+#define RAISE MO(_RAISE)
+#define LOWER MO(_LOWER)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_BASE] = LAYOUT(
+        KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,                                         KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,
+        KC_A,    KC_S,    KC_D,    KC_F,    KC_G,                                         KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN,
+        KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,                                         KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_QUOT,
+                          KC_LBRC, KC_RBRC,                                                        KC_MINS, KC_EQL,
+                                                     SFT_ESC, CTL_BSPC, ALT_SPC, SFT_ENT,
+                                                     KC_TAB,  KC_HOME,  KC_END,  KC_DEL,
+                                                     RAISE,   KC_GRV,   KC_LGUI, LOWER
+    ),
+
+    [_RAISE] = LAYOUT(
+        _______, _______, KC_MS_U, _______, _______,                                      KC_VOLU, _______, KC_UP,   _______, KC_PGUP,
+        _______, KC_MS_L, KC_MS_D, KC_MS_R, _______,                                      KC_MUTE, KC_LEFT, KC_DOWN, KC_RGHT, KC_PGDN,
+        _______, _______, _______, _______, _______,                                      KC_VOLD, KC_SLSH, KC_BSLS, KC_QUES, KC_PIPE,
+                 _______, _______,                                                                          KC_BTN1,  KC_BTN2,
+                                   _______, _______,                                      _______, _______,
+                                                     _______, _______,  _______, _______,
+                                                     _______, _______,  _______, _______
+    ),
+
+    [_LOWER] = LAYOUT(
+        KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,                                        KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,
+        KC_1,    KC_2,    KC_3,    KC_4,    KC_5,                                         KC_6,    KC_7,    KC_8,    KC_9,    KC_0,
+        KC_EXLM, KC_AT,   KC_HASH, KC_DLR,  KC_PERC,                                      KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN,
+                 KC_F11,  KC_F12,                                                                           _______, _______,
+                                   _______, _______,                                      _______, _______,
+                                                     _______, _______,  _______, _______,
+                                                     _______, _______,  _______, _______
+    )
+};
+
+void persistent_default_layer_set(uint16_t default_layer) {
+    eeconfig_update_default_layer(default_layer);
+    default_layer_set(default_layer);
+}
```

Comment:
I suspect that this is just being moved (and git sometimes sucks about this), but this function is deprecated and should be removed.  It's also... not being used here.
```suggestion
```

Reply:
Amended as per [2374fbe](https://github.com/qmk/qmk_firmware/pull/22261/commits/2374fbe326037b8c2c9a1b704621208c12d8f4bc), thanks @drashna 🙂